index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/a1e854484d.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Software Developre | Miles Mosweu</title>
+  </head>
+  <body>
+    <header class="header">
+      <nav class="header__nav container container-pall">
+        <a href="" class="header__logo">Welcome</a>
+        <button class="header__hamburger-btn">
+          <img src="images/Icon - Menu.png" alt="humburger butoon" />
+        </button>
+      </nav>
+    </header>
+    <main class="main">
+      <section id="home" class="intro">
```

Comment:
- [ ] Please consider laying out the contents of this `section` using `flexbox` since this is a project requirement. please see the screenshot below for more details. You can do this the same way you did it on the `nav` :clap: :clap: 
![Screenshot from 2022-12-01 01-37-03](https://user-images.githubusercontent.com/16392046/204923398-807d96ec-ebf1-4b1e-9de0-8bd1b4030a72.png)

Reply:
Thanks for pointing this out. The funny thing is that I was thinking about this exact thing all night.